src/types/label.js

```diff
@@ -13,10 +13,8 @@ export default class LabelAnnotation extends Element {
       const adjust = gap || 0;
       const {x, y, width, height} = this.isBoxVisible() ? this.getProps(['x', 'y', 'width', 'height']) : this.labelRect;
 
-      return mouseX >= x + adjust &&

```

Comment:
I assume you changed these for code climate? I think the old formatting was easier to read.

Reply:
Yes, I prefer the previous one as well. How can I ignore the CC issue?

Reply:
Cc issues are just fyi, we don't require it to pass. There is also possibility to approve the issues in cc (I've sometimes done that)

Reply:
reverted to original format